pkg/authorization/admission/restrictusers/restrictusers.go

```diff
@@ -98,7 +99,12 @@ func (q *restrictUsersAdmission) Admit(a admission.Attributes) (err error) {
 	// We only care about rolebindings and policybindings; ignore anything else.
 	gr := a.GetResource().GroupResource()
 	switch {
-	case authorizationapi.IsResourceOrLegacy("policybindings", gr), authorizationapi.IsResourceOrLegacy("rolebindings", gr):
+	case authorizationapi.IsResourceOrLegacy("policybindings", gr):
```

Comment:
Hmm maybe we do not need this anymore.  During HA upgrade if you talk with a new master about policy bindings it will simply say it does not know what that is.  Old masters will continue to work for policy bindings and origin role bindings.  Old masters also do not allow messing with RBAC by normal users.  @deads2k do you agree that we are safe in dropping this case?

Reply:
We don't care about the policybinding case because new masters (where this admission plugin will be running) will 404 before they get to admission.